packages/terra-graphs-docs/src/terra-dev-site/doc/example/Graphs-Carbon/BarGraph/BasicBarGraphs/TimeseriesWithEventline.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
```

Comment:
Looks like eventline example is not working fine. Can you please take a look at it?

Reply:
fixed here: 54816259a1f0c2401d421315c02ef4dfae784e0f